requirements.txt

```diff
@@ -0,0 +1,2 @@
+tensorflow
+pytest
```

Comment:
Disclaimer:
* First install pytest
* Then deactivate the virtualenv
* Activate the virtualenv again

Now `pytest tests/test_embedding_layer.py ` should run.